backend/apps/slack/management/commands/slack_sync_messages.py

```
@@ -0,0 +1,158 @@
+                if messages:
+                    Message.bulk_save(messages)
+                    self.stdout.write(f"Saved {messages_count} messages")
+
+                cursor = response.get("response_metadata", {}).get("next_cursor")
+                has_more = bool(cursor)
+
+                if delay and has_more:
+                    time.sleep(delay)
+
+            self.stdout.write(
+                self.style.SUCCESS(f"Finished processing {messages_count} messages from {conversation.name}")
```

Ruff (0.11.9)

Line too long (109 > 99)

(E501)